prime-router/settings/organizations.yml

```diff
@@ -1348,6 +1348,207 @@
   jurisdiction: STATE
   stateCode: PA
   receivers:
+    - name: elr-chester-hl7
+      organizationName: pa-phd
+      topic: covid-19
+      customerStatus: active
+      jurisdictionalFilter: [ "matches(ordering_facility_state, PA)", "matches(ordering_facility_county, Chester, Delaware)" ]
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        nameFormat: chester_hl7
+        receivingApplicationName: PA-ELR
+        receivingFacilityName: PADOH
+        suppressHl7Fields: PID-13-1,ORC-23-1,OBR-16
+        replaceValue:
+          ORC-21-2: L
+        valueSetOverrides:
+          hl70189:
+            name: hl70189
+            system: HL7
+            version: 2.9
+            reference: HL7 guidance for ethnicity, modified for PA standards
+            values:
+              - code: H
+                display: Hispanic or Latino
+              - code: NH
+                replaces: N
+                display: Non Hispanic or Latino
+              - code: U
+                display: Unknown
```

Comment:
So we now have the ability to throw a value set override into a translation directly. Then the HL7 serializer will take this and use it instead of the standard value. Is this magic? Maybe a little. It's a tradeoff though because I needed to override the value set either here or I needed to create a schema for the PA HL7 files, and then I'd be adding a new HL7 translation type, one that took custom schemas AND had all the other switches and configuration values, and it seemed like copying all of that over was uglier than doing it this way, even if there is repetition. I'm open to discussing and refactoring though.

Reply:
I think this is a nifty solution, though I suspect at some point you'll want named value set overrides so you can define them once for multiple senders. But that's an easy change to be made later.